src/sleep/battery.c

```diff
@@ -100,15 +104,19 @@ static int siphash24_compress_device_sysattr(sd_device *dev, const char *attr, s
 
         r = sd_device_get_sysattr_value(dev, attr, &x);
         if (r < 0)
-                return log_device_debug_errno(dev, r, "Failed to read '%s' attribute: %m", attr);
+                return log_device_debug_errno(dev, r, "Failed to read '%s' attribute, ignoring: %m", attr);
```

Comment:
This doesn't seem right, we're definitely not ignoring the return value in this function since we're returning an error. I would make these actually ignore the error and return void if that's what you're going for

Reply:
OK. I've dropped `, ignoring`.